src/test/java/com/thealgorithms/maths/SumOfDigitsTest.java

```diff
@@ -1,25 +1,25 @@
-package com.thealgorithms.maths;
```

Comment:
Why change this file?

Reply:
I thought that the file had some conflict I resolved it.
I have reverted it as your request